R/support.R

```diff
@@ -361,18 +361,53 @@ combine_rt <- function(rt, samples) {
   ret
 }
 
+##' Combine Rt estimates from EpiEstim across multiple runs.
+##'
+##' @title Combine Rt estimates from EpiEstim
+##'
+##' @param rt A list of Rt calculations from
+##'   [carehomes_rt_trajectories_epiestim()] (though any Rt calculation that
+##'   confirms to this will work)
+##'
+##' @param samples A list of samples from [carehomes_forecast()]
+##'
+##' @param q A vector of quantiles to return values for
+##'
+##' @return A list of Rt output in the same structure as the first
+##'   element of `rt`. Rt estimates will be aggregated across
+##'   regions (or whatever else you are aggregating on) based on the
+##'   parameters in `samples`.
+##'
+##' @export
+combine_rt_epiestim <- function(rt, samples, q = NULL) {
+  q <- q %||% c(0.025, 0.5, 0.975)
+  ## Ensure all trajectories are the same length
+  ret <- rt[[1L]]
+  if (!("Rt" %in% names(ret))) {
+    stop(paste("rt$Rt missing. Did you forget 'save_all_Rt_sample = TRUE'",
```

Comment:
last issue here - this needs a test that triggers the error

Reply:
done here https://github.com/mrc-ide/sircovid/pull/238/commits/1d29f403c4a1d04058b7eb4c55f88142921642ca